src/fmu/dataio/export/rms/volumetrics.py

```diff
@@ -0,0 +1,253 @@
+            file names, and should not be applied as metadata.
+        classification: Optional. Use 'internal' or 'restricted' (default).
+        workflow: Optional. Information about the work flow; defaulted to
+            'rms volumetrics'.
+        include_zone_maps: Optional. If True, then all the zone maps that are toggled on
+            in the volume job are exported also (currently NOT implemented).
+        include_total_maps: Optional. If True, then all the total maps that are toggled
+            on in the volume job are exported also (currently NOT implemented).
+        include_3dgrid_propererties: Optional. If True, then all the 3D properties
+            (also called parameters) that are toggled on in the volume job are exported
+            also, together with the 3D grid geometry (currently NOT implemented).
+    """
+
+    project: RmsProject
+    grid_name: str
+    volume_job_name: str
+
+    # optional and defaulted
+    config_file: str | Path = "../../fmuconfig/output/global_variables.yml"
+    global_config: dict = field(default_factory=dict)
+    forcefolder: str = ""  # unsure if we shall allow this?
+    tagname: str = "vol"
+    classification: str = "restricted"
+    workflow: str = "rms volumetric run"
+    include_zone_maps: bool = False
+    include_total_maps: bool = False
+    include_3dgrid_properties: bool = False
+
+    # internal storage instance variables
+    _global_config: dict = field(default_factory=dict, init=False)
+    _volume_job: dict = field(default_factory=dict, init=False)
+    _volume_table_name: str = field(default="", init=False)
+    _dataframe: pd.DataFrame = field(default_factory=pd.DataFrame, init=False)
+    _units: str = field(default="metric", init=False)
+
+    @property
+    def dataframe(self) -> pd.DataFrame:
+        return self._dataframe
+
+    @staticmethod
+    def _check_rmsapi_version() -> None:
+        """Check if we are working in a RMS API, and also check RMS versions?"""
+        _logger.debug("Check API version...")
+        if versionparse(rmsapi.__version__) < versionparse("1.7"):
+            raise RuntimeError(
+                "You need at least API version 1.7 (RMS 13.1) to use this function."
+            )
+        _logger.debug("Check API version... DONE")
+
+    def _set_global_config(self) -> None:
+        """Set the global config data by reading the file."""
+        # TODO: This functionality should ideally be in fmu-config.
+        _logger.debug("Set global config...")
+
+        if self.global_config:
+            self._global_config = self.global_config
+            _logger.debug("Set global config (from input dictionary)... DONE!")
+            return
+
+        global_config_path = Path(self.config_file)
+
+        if global_config_path.is_file():
+            self._global_config = yaml_load(global_config_path)
+        else:
+            raise ValueError(f"Cannot find file for global config: {self.config_file}")
```

Comment:
```suggestion
        else:
            raise FileNotFoundError(f"Cannot find file for global config: {self.config_file}")
```
Maybe a `FileNotFoundError` is most appropriate here?